samples/README.md

```diff
@@ -0,0 +1,48 @@
+# ONNX Runtime Samples and Tutorials
+
+Here you will find various samples, tutorials, and reference implementations for using ONNX Runtime. 
+For a list of available dockerfiles and published images to help with getting started, see [this page](../dockerfiles/README.md).
+
+* [Python](#Python)
+* [C#](#C)
+* [C/C++](#CC)
+***
+ 
+## Python
+**Inference only**
```

Comment:
Since these samples are for pre-trained models that are converted to ONNX you can point to AML Training (link: https://github.com/Azure/MachineLearningNotebooks/tree/master/how-to-use-azureml/training) for how to train a model.